java-squid/src/main/java/org/sonar/java/se/SymbolicValue.java

```diff
@@ -22,13 +22,11 @@
 import com.google.common.base.Objects;
 import com.google.common.base.Preconditions;
 import com.google.common.collect.ImmutableList;
-import com.google.common.collect.Maps;
 import org.sonar.java.se.ConstraintManager.BooleanConstraint;
 import org.sonar.java.se.ConstraintManager.NullConstraint;
 
 import java.util.ArrayList;
 import java.util.List;
-import java.util.Map;
 
 public interface SymbolicValue {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move constants to a class or enum. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1214)